packages/matchbox/src/components/CheckboxCard/CheckboxCard.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { CheckBox, CheckBoxOutlineBlank } from '@sparkpost/matchbox-icons';
+import { Box } from '../Box';
+import Group from './Group';
+import { StyledLabel, StyledInput, StyledUnchecked, StyledChecked, StyledHeader } from './styles';
+
+export type CheckboxCardProps = {
+  'data-id'?: string;
+  'data-track'?: string;
+  label?: React.ReactNode;
+  size?: 'small' | 'default';
+} & Omit<React.ComponentPropsWithoutRef<'input'>, 'type' | 'size'>;
+
+const CheckboxCard = React.forwardRef<HTMLInputElement, CheckboxCardProps>(function CheckboxCard(
+  props,
+  userRef,
+) {
+  const {
+    id,
+    checked,
+    children,
+    defaultChecked,
+    disabled,
+    label,
+    name,
+    onChange,
+    onFocus,
+    onBlur,
+    size = 'default',
+    value,
+    'data-id': dataId,
+    'data-track': dataTrack,
+    ...rest
+  } = props;
+
+  const isSmall = size === 'small';
+
+  return (
+    <Box data-id="radio-card">
+      <StyledInput
+        checked={checked}
+        data-id={dataId}
+        data-track={dataTrack}
+        disabled={disabled}
+        defaultChecked={defaultChecked}
+        id={id}
+        name={name}
+        onChange={onChange}
+        onFocus={onFocus}
+        onBlur={onBlur}
+        ref={userRef}
+        type="checkbox"
+        value={value}
+        {...rest}
+      />
+      <StyledLabel as="label" htmlFor={id} $size={size}>
+        <Box display="flex">
+          <Box flex="0 0 auto" width={isSmall ? '200' : '400'}>
+            <Box position="absolute" top={isSmall ? '100' : '500'} left={isSmall ? '200' : '500'}>
+              <StyledUnchecked size="1rem" as={CheckBoxOutlineBlank} />
+              <StyledChecked size="1rem" as={CheckBox} />
+            </Box>
+          </Box>
+          <Box flex="1" pl="300">
+            <StyledHeader
+              data-id="radio-card-header"
+              fontSize={isSmall ? '200' : '300'}
+              lineHeight={isSmall ? '200' : '400'}
```

Comment:
lineHeight here should be `300` not `400`

Reply:
Looks like this is sent to 400 to account for the extra padding in the icons, same as RadioCard

Reply:
Its a bit offset without it
<img width="438" alt="Screen Shot 2022-03-02 at 9 01 08 AM" src="https://user-images.githubusercontent.com/3903325/156376130-8074596e-063f-4bd8-953e-68eb96cf7ee8.png">